src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -1995,9 +2087,14 @@ internal interface I { }
 class B<T> : A<T> where T : A<T>.I
 {
 }";
-            var comp = CreateCompilation(source, parseOptions: TestOptions.Regular7);
+            var comp = CreateCompilation(new[] { source }, parseOptions: TestOptions.Regular7);
             comp.VerifyDiagnostics();
-            comp = CreateCompilation(source, parseOptions: TestOptions.Regular8);
+
+            // PROTOTYPE(NullableReferenceTypes): We need to poison the attribute
+            //var comp = CreateCompilation(new[] { source, NonNullTypesTrue, NonNullTypesAttributesDefinition }, parseOptions: TestOptions.Regular7);
+            //comp.VerifyDiagnostics();
```

Comment:
What is the behavior today? Consider enabling this part of the test unless there is an assert failure.